<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: none;</string>
  </property>
  <widget class="QLabel" name="l_title_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(159, 205, 255);
font: 75 24pt &quot;Times New Roman&quot;;
font: 24pt &quot;Times New Roman Bold&quot;;</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>461</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="l_username">
      <property name="styleSheet">
       <string notr="true">color: rgb(25, 1, 57);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>USERNAME:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtUsername">
      <property name="styleSheet">
       <string notr="true">margin: 8px;
padding: 6px;
border-radius: 10px;
border: 1px solid rgb(25, 1, 57);
</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="l_password">
      <property name="styleSheet">
       <string notr="true">color: rgb(25, 1, 57);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>PASSWORD:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="l_confirmpass">
      <property name="styleSheet">
       <string notr="true">color: rgb(25, 1, 57);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>CONFIRM PASSWORD:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtConfirmPass">
      <property name="styleSheet">
       <string notr="true">margin: 8px;
padding: 6px;
border-radius: 10px;
border: 1px solid rgb(25, 1, 57);
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtPassword">
      <property name="styleSheet">
       <string notr="true">margin: 8px;
padding: 6px;
border-radius: 10px;
border: 1px solid rgb(25, 1, 57);
</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSignup">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>540</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(24, 1, 54);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_accountconfirm">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>590</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Don't you already have an account?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>620</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 10px;
font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Login?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>811</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../OneDrive/Desktop/tempsignup.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_title_1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 24pt &quot;Times New Roman Bold&quot;;
color: rgb(28, 1, 64);</string>
   </property>
   <property name="text">
    <string>EMPLOYEE MANAGEMENT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>l_title_2</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>btnSignup</zorder>
  <zorder>l_accountconfirm</zorder>
  <zorder>btnLogin</zorder>
  <zorder>l_title_1</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
